polishNotation.py

```diff
@@ -0,0 +1,39 @@
+def polishNotation(expression):
+    s = stack()
+    result = 0
+    for exp in range(0, len(expression)-1):
+        exp_list.append(exp)

```

Comment:
NO! This happens |exp|, which is an index, not the actual value. Based on the main loop, this should be:

exp_list.append(expression[exp]).

Was this code working and tested?